corehq/apps/hqadmin/templates/hqadmin/raw_couch.html

```diff
@@ -0,0 +1,34 @@
+{% load hq_shared_tags %}{% load compress %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Doc {{ doc_id }}</title>
+        {% compress css %}
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'style/less/hqstyle-core.less' %}"/>
+        {% endcompress %}
+        {% if less_debug %}
+            <script src="{% static 'style/lib/lessjs/less-1.3.1.min.js' %}"></script>
+            <script>less.watch();</script>
+        {% endif %}
+    </head>
+    <body>
+        <div class="container-fluid">
+            <div class="page-header">
+                <h1>Enter a doc id</h1>
+                <form class="form-horizontal" method="GET" action="">
+                    <input name="id" type="text" style="width: 30em"/>
```

Comment:
another nit, but i think if you used the b3 form classes you probably wouldn't have to use the inline style http://getbootstrap.com/css/#forms